### ISSUE TITLE
Please refactor to es6 and add to bower 

### ISSUE THREAD

**Issue by DDeme** [Flo-Schield-Bobby/colormix#4]

**Comment by flo-sch** [comment]
Yep, that's in the plans, actually the refactoring to ES6 is already done ;
However I didn't find the good way to publish it available as :
- standalone JS library,
- ES5 module
- ES6

Still need to tweak the compilation of ES6 in order to make it available this way.

**Comment by DDeme** [comment]
May  I help you ?

**Comment by flo-sch** [comment]
Any contribution is very welcome, I am currently not available to work on this before november 16th sorry ;
But if you have an idea of the process and would like to make a PR, I can take the time to look at it!
